akka-http-proxy/src/test/scala/com/pagerduty/akka/http/proxy/HttpProxySpec.scala

```diff
@@ -77,5 +83,17 @@ class HttpProxySpec extends FreeSpecLike with Matchers {
 
       p.request(HttpRequest(), upstream)
     }
+
+    "can write headers from the request to the response" in {
```

Comment:
This is a good test. Wording it to express the more general functionality would be slightly better though: "transforms responses for an upstream".